### PR TITLE
Add configuration files

### PR DESCRIPTION
**Pull request by mvsusp**
# Add configuration files

I'm closing https://github.com/aws/sagemaker-container-support/pull/23 since both have the setup.py file changes. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by laurenyu** [comment]
can you fix the build error?

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    teamcity-messages
+    awslogs
+
+[testenv:flake8]
+basepython = python
+deps =
+    flake8
+    pep8-naming
+    flake8-import-order
+commands = flake8
```
Comment (laurenyu) [inline tox.ini]: don't forget a newline at the end of the file

Comment (mvsusp) [reply tox.ini]: done

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+def read(file_name):
+    return open(os.path.join(os.path.dirname(__file__), file_name)).read()
+
+
+
+setup(
+    name='sagemaker-container-support',
+    version='1.0',
```
Comment (winstonaws) [inline setup.py]: Version should be 2.0.0

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+from setuptools import find_packages, setup
+
+
+def read(file_name):
+    return open(os.path.join(os.path.dirname(__file__), file_name)).read()
+
+
+
+setup(
+    name='sagemaker-container-support',
```
Comment (winstonaws) [inline setup.py]: Name should match our existing name: https://github.com/aws/sagemaker-container-support/blob/master/setup.py#L16

so sagemaker_container_support

I know it doesn't follow python convention, but I think if we're keeping the same name, then continuity is important. (If we decide to change the name, everything changes, of course.)

**Comment by mvsusp** [comment]
@laurenyu @nadiaya , Yuan will work in the name. Using the old name in this PR.

**Pull request merged**